master/internal/api_experiment.go

```diff
@@ -51,6 +53,65 @@ import (
 
 var experimentsAddr = actor.Addr("experiments")
 
+// Catches information on active running experiments.
+type experimentAllocation struct {
+	Job      model.JobID
+	Pulling  bool
+	Running  bool
+	Starting bool
+}
+
+func (a *apiServer) enrichExperimentState(experiments []*experimentv1.Experiment) (
```

Comment:
this can just return an error now. and what i meant was to make this
```suggestion
func (a *apiServer) enrichExperimentState(experiments ...*experimentv1.Experiment) (
```